pipeline/recon-pipeline.py

```diff
@@ -235,6 +235,16 @@ def do_scan(self, args):
         # luigi --module pipeline.recon.web.webanalyze WebanalyzeScan --target-file tesla --top-ports 1000 --interface eth0
         command = ["luigi", "--module", scans.get(args.scantype)[0]]
 
+        if args.target and args.target_file:
+            self.async_alert(style(f"[!] You can't use the --target and --target-file options in the same scan!"))
+            return
+        elif args.target:
+            with open("tmp_tgt", "w") as file:
+                file.write(args.target)
+                args.__statement__.arg_list[args.__statement__.arg_list.index("--target") + 1] = os.path.abspath(
+                    "tmp_tgt"
```

Comment:
I like the idea of a temp file, but i'm not crazy about it being hard-coded.  I'm also not a fan of it being dropped in the user's cwd.  Take a look at [mkstemp](https://docs.python.org/3/library/tempfile.html#tempfile.mkstemp).  That will drop it in /tmp and give it a random name.  

Should probably be good stewards and do a check around [here](https://github.com/GreaterGoodest/recon-pipeline/blob/add-target/pipeline/recon-pipeline.py#L271) for `args.target` and if found clean up the file created by `mkstemp`.